bitsail-common/src/main/java/com/bytedance/bitsail/common/typeinfo/TypeInfos.java

```diff
@@ -64,6 +64,8 @@ public final class TypeInfos {
       new BasicTypeInfo<>(LocalTime.class);
   public static final TypeInfo<LocalDateTime> LOCAL_DATE_TIME_TYPE_INFO =
       new BasicTypeInfo<>(LocalDateTime.class);
+  public static final TypeInfo<byte[]> BINARY_TYPE_INFO =
```

Comment:
Actually we have `binary` type already.
You can directly use `BasicArrayTypeInfo.BINARY_TYPE_INFO`.
So I suggest this one can be removed.